src/schemas/tasks.schema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const taskSchema = z.object({
+  id: z.string(),
```

Comment:
i think it is good to set as task_id, to be able to identify better in any context, and also, to be an uuid
```suggestion
  task_id: z.string().uuid(),
```